### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by sachhm**
@Ya-Jun-Liu, would you please be able to look at my code? I'd like comments on my class design if that's all good. Thanks 😸

### CODE REVIEW

File: prac_06/car.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        """Return Car as string"""
```
Comment (Ya-Jun-Liu) [inline prac_06/car.py:17]: It might be a good idea to put a full stop at the end.

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        """Return Guitar as string"""
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
+    def get_age(self):
+        """Get age of Guitar given year"""
+        return CURRENT_YEAR - self.year
+
+    def is_vintage(self):
+        """Determine if a guitar is vintage given a threshold"""
+        return self.get_age() > VINTAGE_THRESHOLD
```
Comment (Ya-Jun-Liu) [inline prac_06/guitar.py:36]: Maybe forgot to include 50?

File: prac_06/guitar_test.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Expected Time: 10 Minutes
+Actual Time: 4 Minutes
+"""
+from prac_06.guitar import Guitar
+
+gibson = Guitar("Gibson L-5 CES", 1922, 16035.40)
+stratocaster = Guitar("Fender Stratocaster American Pro", 2021, 2510.28)
+
+print(gibson)
+print(f"Age: {gibson.get_age()}")  # Expected 101, got 101
```
Comment (Ya-Jun-Liu) [inline prac_06/guitar_test.py:14]: if the gibson.get_age() returns the wrong value, could we detect from the printout?

File: prac_06/guitars.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+"""
+CP1404/CP5632 Practical
+Guitars
+
+Expected Time: 40 Minutes
+Actual Time:  20 Minutes
```
Comment (Ya-Jun-Liu) [inline prac_06/guitars.py:6]: Nice! You are more efficient than anticipated.

Comment (sachhm) [reply prac_06/guitars.py:6]: Thanks :)

File: prac_06/car.py (hunk 1 above)
Comment (sachhm) [reply prac_06/car.py:17]: Added 👍

File: prac_06/guitar_test.py (hunk 3 above)
Comment (sachhm) [reply prac_06/guitar_test.py:14]: You could if you knew what the expected value was. If it returns the wrong value, the printout would reflect that.

**Pull request merged**